File: Java_Codes/10. Packages & Access Modifiers/mypackage/Employee.java
package mypackage; // User-defined package

import java.time.LocalDate; // Built-in package

// Employee class to store employee details
public class Employee {
    private String name;
    private int age;
    private LocalDate joiningDate;

    public Employee(String name, int age, LocalDate joiningDate) {
        this.name = name;
        this.age = age;
        this.joiningDate = joiningDate;
    }

    // Method to return employee details as a formatted string
    public String getEmployeeDetails() {
        return name + ", " + age + ", Joined on: " + joiningDate;
    }
}

File: Python_Codes/10. Packages & Access Modifiers/mypackage/Employee.py
# employee.py (This would be your user-defined module, similar to a Java package)

from datetime import date

class Employee:
    def __init__(self, name: str, age: int, joining_date: date):
        self.name = name
        self.age = age
        self.joining_date = joining_date

    def get_employee_details(self) -> str:
        return f"{self.name}, {self.age}, Joined on: {self.joining_date}"
